less/navs.less

```diff
@@ -149,10 +149,41 @@
 .nav-stacked {
   > li {
     float: none;
+
+    > a {
+      font-size: @font-size-base;
```

Comment:
for some reasons we use 14px font-size in our bootstrap theme
you can have a look at nav.less line 24 "font-size: @font-size-small;"